package.json

```diff
@@ -34,6 +34,7 @@
   },
   "dependencies": {
     "audioworklet-polyfill": "^1.1.2",
+    "p5": "^x.0.0",
```

Comment:
it seems like this is the same as "latest", which might be a bit easier to read?

Also, funny that it installs a file that says "0.10.2" at the top with this, and with "latest" too. That seems to be the latest version in npm and we should publish p5.js with "1.0.0" !